Upgrade chatbot example to langchain 0.3

miguelgrinberg opened the pull request:
I have updated the chatbot RAG example to use the recently released langchain 0.3.

alizenhom commented:
does the chatbot rag example work without this PR?

i have tried to spin up the chatbot RAG app, followed the documentation step by step but there is a dependency conflict happening


i always get this error when trying to run `flask run`

`ModuleNotFoundError: No module named 'langchain_elasticsearch'`

also `langchain-community` is missing from dependecies

miguelgrinberg commented:
I believe the example is currently broken, at least I got a report of dependency issues, which is why I looked into this.

The langchain-community package used to be part of the core langchain, and it isn't anymore. I have added it as a top-level dependency in this PR.

I'm not sure why you get errors for langchain-elasticsearch. This package was also split from core langchain a while ago, but it is explicitly listed in the requirements already.
